src/System.Data.Common/dir.props

```diff
@@ -5,7 +5,7 @@
     <AssemblyVersion>4.2.0.0</AssemblyVersion>
     <AssemblyKey>MSFT</AssemblyKey>
     <IsNETCoreApp>true</IsNETCoreApp>
-    <IsDesktopFacade>true</IsDesktopFacade>
     <IsUAP>true</IsUAP>
+    <IsNetFxNETStandard>true</IsNetFxNETStandard>
```

Comment:
Anything that was needed to fill in types missing from https://github.com/dotnet/corefx/pull/18246/files/fe4dded7409185857ab988374b9a86094a7c4d38#diff-798d761ba1611732f1e4a2291a2e584f.

For things that aren't in, we just pick up the nestandard.dll-based-facade from the NETStandard.Library package.